### PR TITLE
Service Catalog test

### PR DESCRIPTION
**Pull request by Skyterix1991**

### CODE REVIEW

File: kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/ServiceCatalogCrudTest.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+import io.fabric8.servicecatalog.client.DefaultServiceCatalogClient;
+import io.fabric8.servicecatalog.client.ServiceCatalogClient;
+import org.junit.Rule;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.migrationsupport.rules.EnableRuleMigrationSupport;
+
+import static org.junit.Assert.*;
```
Comment (kunal-kushwaha) [inline kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/ServiceCatalogCrudTest.java]: Remove wildcard imports.

**Comment by kunal-kushwaha** [comment]
@rohanKanojia : Can you please provide your feedback when you get time?

File: kubernetes-tests/pom.xml [hunk 2]
```diff
@@ -76,6 +76,12 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>servicecatalog-client</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
```
Comment (kunal-kushwaha) [inline kubernetes-tests/pom.xml]: I think you need to add this as well : 
```
<dependency>
      <groupId>io.fabric8</groupId>
      <artifactId>servicecatalog-server-mock</artifactId>
      <version>${project.version}</version>
      <scope>test</scope>
</dependency>
```

**Comment by RaghavAwasthi** [comment]
@Skyterix1991 Can you squash your commits.

**Comment by kunal-kushwaha** [comment]
@Skyterix1991 : Can you please add a line in CHANGELOG.md for this test?

**Comment by rohanKanojia** [comment]
ok to test

**Comment by kunal-kushwaha** [comment]
@rohanKanojia : Can you retest this please?

**Comment by rohanKanojia** [comment]
[merge]

**Pull request merged**